config.py

```diff
@@ -2,8 +2,9 @@
 basedir = os.path.abspath(os.path.dirname(__file__))
 
 class Config(object):
-    SECRET_KEY = ''
+    SECRET_KEY = 'test'
```

Comment:
I'd prefer to leave unset and have operator fill manually if intending to use with docker

```suggestion
    SECRET_KEY = ''
```